packages/jaeger-ui/src/components/Monitor/ServicesView/index.tsx

```diff
@@ -287,6 +304,31 @@ export class MonitorATMServicesViewImpl extends React.PureComponent<TPropsWithIn
                 ))}
               </Field>
             </Col>
+            <Col span={6}>
+              <h2 className="span-kind-selector-header">Choose kind</h2>
+              <Field
+                name="spanKind"
+                component={reduxFormFieldAdapter({ AntInputComponent: SearchableSelect })}
+                placeholder="Select A Span Kind"
+                onChange={(e, value: string) => {
+                  const { label } = spanKindOptions.find(option => option.value === value)!;
+                  trackSelectSpanKind(label);
+                }}
+                props={{
+                  className: 'span-kind-selector',
+                  defaultValue: spanKindOptions[0],
```

Comment:
client?

Reply:
also, better not to depend on the array order.

Reply:
👍🏼 Done in: [46989de](https://github.com/jaegertracing/jaeger-ui/pull/2012/commits/46989deae9944f243b7c267fe197c64e23246dc7)